contents/careers.md

```diff
@@ -57,15 +66,22 @@ Putting things in writing helps us clarify our own ideas, as well as allow other
 
 We're young, but we're already [backed](/handbook/strategy/investors) by many of the best investors in the world and have [raised millions of dollars in funding](/blog/raising-3m-for-os).
 
-This includes YCombinator, Solomon Hykes (Founder of Docker), David Cramer (Founder/CEO of Sentry), Adam Goldstein (Founder of Hipmunk), and [many more](/handbook/strategy/investors)!
+This includes YCombinator, Solomon Hykes (Founder of Docker), David Cramer (Founder/CEO of Sentry), Adam Goldstein (Founder of Hipmunk), and [many more](/handbook/strategy/investors).
+
+## The Process
+
+We will schedule 2 to 3 short calls with our founders and team. These will focus on two key areas:
 
-## A Fair Hiring Process
+* Ensuring you will be a great cultural fit
+* Aptitude for the role
 
-What we truly care about is how well you can contribute, as well as how good of a fit you are for the role. As such, the best way to establish this is by getting you to work with us!
+If you succeed with the above, we will set you up with a *PostHog SuperDay*.
 
-Following a few calls with our founders and team, if we think you're a good fit, we'll hire you as a contractor for 1-5 days to do some work with us. You will get paid normal contracting rates (i.e. above what your pro rata salary would be per day) and work on some real needs we have at PostHog.
+This is a *paid* day of working with us, which we will fit around your schedule. Getting hired by us is extremely challenging, and this will be a big day. You'll need to be fully engaged with a steady supply of tea, coffee and biscuits.
```

Comment:
Just confirming here that we want to use the strong term 'extremely challenging'?

Reply:
Done